pom.xml

```diff
@@ -62,6 +62,7 @@
         <dependency>
             <groupId>org.springframework.security</groupId>
             <artifactId>spring-security-crypto</artifactId>
+            <version>5.3.0.RELEASE</version>
```

Comment:
This is overriding the managed version. However, Spring Boot does not provide a version including this one yet. 
It'll be worth waiting for a new release of `spring-boot-starter-parent` and then update our [parent pom](https://github.com/companieshouse/spring-boot-parent-pom) and upgrade it here

Reply:
`spring-boot-starter-parent` 2.3.0.RELEASE is now out and it uses 5.3.2.RELEASE of `spring-security-crypto` so we would need to use that one instead of overriding this managed version